treeherder/perf/alerts.py

```diff
@@ -106,7 +104,7 @@ def generate_new_alerts_in_series(signature):
                 if t_value == float('inf'):
                     t_value = 1000
 
-                a = PerformanceAlert.objects.create(
+                data = PerformanceAlert.objects.create(
```

Comment:
This should not be changed, I only wanted you to change variables named "a" which formerly referred to the Analyzer class, which this is not an instance of.